pyriemann/classification.py

```diff
@@ -903,4 +902,106 @@ def predict_proba(self, X):
         prob : ndarray, shape (n_matrices, n_classes)
             Probabilities for each class.
         """
-        return softmax(-self._predict_distances(X)**2)
+        return softmax(-self._predict_distances(X) ** 2)
+
+
+def class_distinctiveness(X, y, return_num_denom=False):
+    r"""Measure class separability between classes on the manifold.
+
+    For two class problem, the class distinctiveness between class A
+    and B on the manifold is quantified as:
+
+    .. math::
+        \mathrm{classDis}(A, B) = \frac{\delta_{R}
+                       \left(\bar{C}^{A}, \bar{C}^{B}\right)}
+                       {\frac{1}{2}
+                       \left( \sigma_{C^{A}} + \sigma_{C^{B}}
+                       \right)}
+
+    where :math:`\bar{C}^{K}` and :math:`\sigma_{C^{K}}` are
+    respectively the mean and the mean absolute deviation of the
+    covariance matrices from class K.
+
+    For more than two classes, it is quantified as:
+
+    .. math::
+        \mathrm{classDis}\left(\left\{A_{i}\right\}\right) =
+        \frac{\sum_{i=1}^{N c} \delta_{R}\left(\bar{C}^{A_{i}},
+        \overline{\bar{C}}^{A}\right)}{\sum_{i=1}^{N c} \sigma_{C^{A_{i}}}}
+
+    where :math:`\overline{\bar{C}}^{A}` is the average of the mean covariance
+    matrices of all classes.
```

Comment:
```suggestion
        \overline{C}\right)}{\sum_{i=1}^{N c} \sigma_{C^{A_{i}}}}

    where :math:`\overline{C}` is the average of the mean covariance
    matrices of all :math:`N c` classes.
```
A proposition to simplify the notation

Reply:
Since argument of input matrices is called X, in equations, I would rename matrices C into X.

Reply:
I fixed all C to X. Please confirm it.